carbonmark-api/src/models/Listing.model.ts

```diff
@@ -54,4 +54,12 @@ export const ListingModel = Type.Object(
   }
 );
 
+export const ListingWithTokenSymbolModel = Type.Intersect([
+  ListingModel,
+  Type.Object({
+    symbol: Type.String({ description: "Symbol of the token" }),
+  }),
+]);
+
```

Comment:
We should have only one listing model with the token symbol as an optional attribute

Reply:
done in [d88d0f9](https://github.com/KlimaDAO/klimadao/pull/2046/commits/d88d0f9a0bc0ec5b3d174f7c5c312a6a5b3719f1)